src/kudu/consensus/raft_consensus.cc

```diff
@@ -181,6 +181,9 @@ DEFINE_bool(enable_flexi_raft, false,
             "Enables flexi raft mode. All the configurations need to be already"
             " present and setup before the flag can be enabled.");
 
+DEFINE_bool(withold_votes_for_tests, false,
```

Comment:
the approach of global variable does not work well, because we run multiple rafts in one process.
but for flexi-raft testing, you would likely want to withhold votes on specific instances.

Reply:
Discussed over chat. Also removed merlin from the message.